src/fides/api/util/tcf/tc_model.py

```diff
@@ -0,0 +1,359 @@
+import json
+import re
+from datetime import datetime
+from os.path import dirname, join
+from typing import Dict, List, Optional, Tuple
+
+from fideslang.models import LegalBasisForProcessingEnum
+from pydantic import Field, NonNegativeInt, PositiveInt, validator
+
+from fides.api.models.privacy_notice import UserConsentPreference
+from fides.api.schemas.base_class import FidesSchema
+from fides.api.schemas.tcf import TCFFeatureRecord, TCFPurposeRecord, TCFVendorRecord
+from fides.api.util.tcf_util import TCFExperienceContents
+from fides.config.helpers import load_file
+
+# TODO replace with a load_gvl() call after https://github.com/ethyca/fides/pull/4143 is merged
+GVL_JSON_PATH = join(
+    dirname(__file__),
+    "../../../../../clients/fides-js/src/lib/tcf",
+    "gvl.json",
+)
```

Comment:
Just waiting on https://github.com/ethyca/fides/pull/4143 to get merged, this is leftover from the POC.

Reply:
OK done!